master/src/main/scala/org/apache/celeborn/service/deploy/master/MasterArguments.scala

```diff
@@ -44,7 +44,9 @@ class MasterArguments(args: Array[String], conf: CelebornConf) {
     val localNode = clusterInfo.localNode
     _host = _host.orElse(Some(conf.haMasterNodeHost(localNode.nodeId)))
     _port = _port.orElse(Some(conf.haMasterNodePort(localNode.nodeId)))
-    _internalPort = _internalPort.orElse(Some(conf.haMasterNodeInternalPort(localNode.nodeId)))
+    _internalPort = _internalPort.orElse { 
```

Comment:
```suggestion
    _internalPort = _internalPort.orElse {
```

Reply:
my bad, there is a extra space at the end

Reply:
+1